.rubocop.yml

```diff
@@ -19,6 +19,8 @@ Metrics/BlockLength:
   AllowedMethods:
     - describe
   Max: 30
+Metrics/CyclomaticComplexity:
+  Enabled: false
```

Comment:
- [ ] Please instead of modifying the linter configuration try to correct the offenses in your code. Linters serve as a way of having a consistent code format and ensuring some best practices. Kindly consider fixing this! 🤓 🔧